src/sage/groups/braid.py

```diff
@@ -3407,7 +3519,7 @@ def BraidGroup(n=None, names='s'):
     - ``n`` -- integer or ``None`` (default). The number of
       strands. If not specified the ``names`` are counted and the
       group is assumed to have one more strand than generators.
-
++
```

Comment:
Please revert this entire change.

Reply:
A blank line appears after `class` in other parts

Reply:
I do not understand *this entire change*. There was an extra `+` due to my fingers, removed.

Reply:
Blanklines between inputs are unnecessary and evil IMO. I don't want to add them anywhere, especially where it wasn't before. (It also helps prevent merge conflicts as it is not where you are making changes. Many of your other changes fall under this too, but I don't disapprove of them.)

Reply:
I reverted both changes, but actually in this file for most classes there is a blank line (not introduced by me)